src/test/validation_chainstatemanager_tests.cpp

```diff
@@ -387,82 +387,57 @@ BOOST_FIXTURE_TEST_CASE(chainstatemanager_activate_snapshot, SnapshotTestSetup)
     this->SetupSnapshot();
 }
 
-//! Test LoadBlockIndex behavior when multiple chainstates are in use.
-//!
-//! - First, verfiy that setBlockIndexCandidates is as expected when using a single,
-//!   fully-validating chainstate.
-//!
-//! - Then mark a region of the chain BLOCK_ASSUMED_VALID and introduce a second chainstate
-//!   that will tolerate assumed-valid blocks. Run LoadBlockIndex() and ensure that the first
-//!   chainstate only contains fully validated blocks and the other chainstate contains all blocks,
-//!   even those assumed-valid.
-//!
-BOOST_FIXTURE_TEST_CASE(chainstatemanager_loadblockindex, TestChain100Setup)
```

Comment:
In commit "test: add testcases for snapshot initialization" (5ff0d2f7e78a40225a1eb9aa7bbf5c358d94bf48)

Something wonky is going on in this commit. It seems like some rebase error. This is dropping the `chainstatemanager_loadblockindex` test, and replacing it with an exact copy of the `chainstatemanager_snapshot_init` below, which now appears twice (and is edited in later commit 65ad0860ffd9441b005ac4bc240f596d9a81836f to become a different chainstatemanager_snapshot_completion test)

Reply:
@ryanofsky yeah, thanks - not sure what happened there. Should be fixed.